<launch>
	<node pkg="sunray_comm" type="mavlink_node" name="mavlink_node" output="screen">
		<rosparam command="load" file="$(find sunray_comm)/launch/params.yaml" />
	</node>
</launch>

